core-java/src/main/java/com/baeldung/tostring/CustomerS5.java

```diff
@@ -0,0 +1,49 @@
+package com.baeldung.tostring;
+
+import java.util.List;
+
+import org.apache.commons.lang.builder.ReflectionToStringBuilder;
+
+public class CustomerS5  {
+    private String firstName;
```

Comment:
There is a lot of duplicate code across all of these customer classes. Can't we have on parent `Customer` class and then extend it for all of these other classes? Also, lets name these classes a little bit better to describe the differences of the `toString` methods in each (i.e. this class could be called `CustomerReflectionToString`)